### PR TITLE
Fix issue #1218: Arrange, Act, Assert pattern

### PR DESCRIPTION
**Pull request by npczwh**
Pull request title

- Implemented Arrange, Act, Assert pattern for #1218

Pull request description

- Ref [Arrange, Act, Assert: What is AAA Testing?](https://blog.ncrunch.net/post/arrange-act-assert-aaa-testing.aspx)

### CODE REVIEW

File: arrange-act-assert/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: pattern
+title: Arrange/Act/Assert
+folder: arrange-act-assert
+permalink: /patterns/arrange-act-assert/
+categories: Idiom
+tags:
+ - Unit Test
```
Comment (iluwatar) [inline arrange-act-assert/README.md:8]: We don't have an existing tag yet, but I would propose to use `Testing` here. Then we need to amend file https://github.com/iluwatar/java-design-patterns-web/blob/master/_data/tags.yml. See https://github.com/iluwatar/java-design-patterns/wiki/07.-Categories-and-Tags.

**Comment by npczwh** [comment]
@iluwatar Thank you for your comments. I have done the required changes. I also created a PR in java-design-patterns-web to add the new tag (https://github.com/iluwatar/java-design-patterns-web/pull/12).
By the way, did we stop using Travis CI?  I can't find any checks in this PR.

File: arrange-act-assert/README.md (hunk 1 above)
Comment (npczwh) [reply arrange-act-assert/README.md]: Changed in 882b569

**Comment by iluwatar** [comment]
Thanks @npczwh for the new pattern, great job! :+1: 

I'm looking into the Travis issue why it hasn't built the PRs originated from forks.

**Pull request merged**